brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/value/ResValueFactory.java

```diff
@@ -108,7 +108,7 @@ public ResBagValue bagFactory(int parent, Duo<Integer, ResScalarValue>[] items,
             return new ResPluralsValue(parentVal, items);
         }
 
-        if (ResTypeSpec.RES_TYPE_NAME_STYLES.equals(resTypeName)) {
+        if (resTypeName.equals(resTypeName)) {
```

Comment:
Not sure about this. This makes this 100% true every-time no? So what about the ResAttr detection below this? What it looks like you are trying to do is default an unknown res type to styles, but not sure if that is the correct go here.